assets/javascripts/job_next_previous.js

```diff
@@ -86,18 +86,22 @@ function renderJobResults(data, type, row) {
         if (count++) {
             var more = row.failedmodules.length - count + 1;
             if (more > 0 && limit < 12) {
-                html += '+' + more;
+                html += '<a title="';
+                for (var j = i; j< row.failedmodules.length ; j++) {
+                    html += "- " + htmlEscape(row.failedmodules[j]) + "\n";
```

Comment:
Can we please use jQuery here instead of more string concatenation and escaping which isn't even unit tested?
I realize you might not want to rewrite the code, so getting the HTML from jQuery would be totally fine with me.

Reply:
I don't think it is possible to return DOM elements here. In my opinion it also does not make sense to use jQuery to create DOM elements just to serialize them after all.